src/main/java/lotto/domain/lotto/LottoNumber.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain.lotto;
+
+import java.util.Objects;
+
+import static lotto.Validator.isInRange;
+
+public class LottoNumber {
+    public static final int MIN_LOTTO_NUMBER = 1;
+    public static final int MAX_LOTTO_NUMBER = 45;
+    private static final String WRONG_LOTTO_NUMBER_MESSAGE = "(%d): 잘못된 번호입니다.";
+
+    private final int lottoNumber;
+
+    private LottoNumber(int lottoNumber) {
+        validateLottoNumberRange(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    private void validateLottoNumberRange(int lottoNumber) {
+        if (!isInRange(lottoNumber, MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)) {
+            throw new IllegalArgumentException(String.format(WRONG_LOTTO_NUMBER_MESSAGE, lottoNumber));
```

Comment:
Nit: 사용자 정의 예외 클래스를 정의하여 예외 메시지를 관리해보는 것도 좋을 것 같습니다.
예외 메시지도 결국 사용자에게 보여지는 출력과 관련된 부분인데, 이 메시지를 수정해야 하는 상황이 왔을 때
핵심 도메인인 `LottoNumber` 대신에 메시지를 관리하는 예외 클래스만 수정하면 되지 않을까 싶네요.
```suggestion
            throw new InvalidLottoNumberException(lottoNumber);
``